lib/fastlane/plugin/release_actions/actions/set_constant_value.rb

```diff
@@ -0,0 +1,69 @@
+module Fastlane
+  module Actions
+    class SetConstantValueAction < Action
+      def self.run(params)
+        file = params[:file]
+        constant = params[:constant]
+        value = params[:value]
+        
+        file_contents = File.read(file)
+
+        CONSTANT_VERSION = /(?<=#{constant})(\s*=\s*["'].*)/
```

Comment:
Couple of nits here:

1. This could be used for anything and not necessarily just versions, so perhaps we could generalize the name?
1. While we're at it... this is actually matching a global, not a constant, in its use. The pattern above will match local and global variables, as well as instance variables, and constants. I'm not sure I have a better suggestion for a name. Perhaps declaration? Or variable? (which is weird, given it could be a constant, but idk).

Reply:
Could just do `key`